packages/usdk/packages/upstreet-agent/packages/react-agents/components/plugins/discord.tsx

```diff
@@ -11,6 +11,7 @@ import {
 export const Discord: React.FC<DiscordProps> = (props: DiscordProps) => {
   const {
     token,
+    appId,
```

Comment:
Can the appId be loaded from the `token`?

Reply:
The appId cannot be directly extracted from the token, for dymanically getting the appId, we can use the server to send back the applicationId/bot's user via client.user, that would require ensuring the appId is available before creating conversations hence i prefer the user itself setting the "appId" for the bot